bigsea-scaler/test/service/api/controller/test_proportional_alarm.py

```diff
@@ -40,8 +40,13 @@ def setUp(self):
         self.actuator = KVM_Actuator(self.instance_locator, self.remote_kvm)
 
         self.conservative_factor = 1
-        self.heuristic_options = {"heuristic_name":"error_proportional",
+        self.factor_up = 1
+        self.factor_down = 0.5
+        self.heuristic_options_1 = {"heuristic_name":"error_proportional",
```

Comment:
careful with this heuristic_options names: maybe self.heuristics_options_error_proportional (or self.ho_error_proportional) and the following too.